resources/kubernetes/operator/helm/values.yaml

```diff
@@ -44,60 +47,68 @@ clusterRole:
 # Cluster Name is detected automatically for EKS/GKE/AKS
 # clusterName: myClusterName
 collectors:
+  #  Cluster is a K8s deployment EDOT collector focused on gathering telemetry
+  #  at the cluster level (Kubernetes Events and cluster metrics).
   cluster:
     config:
       exporters:
+        # [Debug exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/debugexporter/README.md)
         debug:
-          verbosity: basic
+          verbosity: basic # Options: basic, detailed. Choose verbosity level for debug logs.
+         # [Elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/elasticsearchexporter/README.md)
         elasticsearch/otel:
           endpoints:
-          - ${env:ELASTIC_ENDPOINT}
-          api_key: ${env:ELASTIC_API_KEY}
+          - ${env:ELASTIC_ENDPOINT}  # API key for Elasticsearch authentication.
+          api_key: ${env:ELASTIC_API_KEY} # List of Elasticsearch endpoints.
```

Comment:
Should be the other way around, this comment should go above to env:ELASTIC_ENDPOINT and vs :)